src/mantine-dates/src/components/TimeInput/TimeInput.tsx

```diff
@@ -10,11 +10,16 @@ import {
   __InputStylesNames,
   ElementProps,
 } from '@mantine/core';
+import dayjs from 'dayjs';
 import classes from './TimeInput.module.css';
 
 export interface TimeInputProps extends InputBaseProps, ElementProps<'input', 'size'> {
   /** Determines whether seconds input should be rendered */
   withSeconds?: boolean;
+  /** Minimum possible time */
```

Comment:
There must be a new line between previous prop and new prop description, see other components for reference.

Reply:
New lines inserted between props